### PR TITLE
fix(chart): helm-git support

### PR DESCRIPTION
**Pull request by bdellegrazie**
# Changes

Modify the location of the helm chart to match the default expected by the helm/helm-release github action.
The default location (`charts/<name-of-chart>`) is directly supported by the [helm-git](https://github.com/aslafy-z/helm-git) plugin so the chart can be used without publication too.

# Release Notes

```release-note
Move the helm chart to the standard location (`charts/tekton-operator`) so that it can be consumed directly from the git repository via the [helm-git](https://github.com/aslafy-z/helm-git) plugin.
```

### CODE REVIEW

File: charts/tekton-operator/README.md [hunk 1]
```diff
@@ -1,23 +1,29 @@
...
+Once this chart is published it can be installed directly, until then [helm-git](https://github.com/aslafy-z/helm-git) can be used.
 
 ## TL;DR
 
+Install [helm-git](https://github.com/aslafy-z/helm-git) plugin.
+
 ```sh
-git clone https://github.com/tektoncd/operator.git
+# Add the repo to helm (typically use a tag rather than master):
+helm repo add tektoncd "git+https://github.com/tektoncd/operator@charts?ref=master"
```
Comment (vdemeester) [inline charts/tekton-operator/README.md:12]: Should probably be `main` instead of `master` for the ref.

Comment (bdellegrazie) [reply charts/tekton-operator/README.md]: Fixed - thank you

**Comment by piyush-garg** [comment]
/lgtm

@bdellegrazie can you please squash the commits into 1

**Comment by bdellegrazie** [comment]
@piyush-garg : squashed as requested.

**Comment by piyush-garg** [comment]
/lgtm
cc @vdemeester @jkandasa

**Comment by jkandasa** [comment]
/lgtm 
cc @vdemeester

**Review by jkandasa — APPROVED** [review]
